_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -68889,20 +68911,20 @@
 "cGJ" = (
 /obj/effect/decal/cleanable/dirt,
 /obj/machinery/door/airlock{
-	name = "Toilet Unit"
+	id_tag = "Toilet1";
+	name = "Toilet Unit 1"
 	},
 /obj/effect/turf_decal/tile/neutral{
 	dir = 1
 	},
 /obj/effect/turf_decal/tile/neutral{
 	dir = 4
 	},
-/turf/open/floor/plasteel{
-	heat_capacity = 1e+006
-	},
```

Comment:
This is a code improvement and a fix

Reply:
Oh yeah that popped up on icebox's bathroom tiles as well, almost tried to make a new one. Not sure what the deal with that is. Strongdmm having some sort of moment.